### DISCUSSION TITLE
how to prevent greedy tag replacement

### DISCUSSION THREAD

**Discussion by DMazona** [handlebars-lang/handlebars.js#1921] [Q&A] [answered]
I can't seem to find the answer for this in examples and documentation.
In practice, I have a template that includes three braces '<node>hello_{{{var}}}</node>' where I only want the inner '{{var}}' replaced, so that then end result is '<node>hello_{world}</node>', but the end result instead is '<node>hello_world</node>', probably because '{{{' is the escape syntax for mustache. Is there a way to change the escape syntax tag? 
In my case, I can't switch the tag for something else, because of a compatibility issue.

**Comment by jaylinski** [comment, marked as answer]
You can use white space control to achieve this:

```
hello_{ {{~var~}} }
```

Demo: https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22hello_%7B%20%7B%7B~var~%7D%7D%20%7D%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%5Cn%20%20var%3A%20%5C%22world%5C%22%5Cn%7D%5Cn%22%2C%22output%22%3A%22hello_%7Bworld%7D%22%2C%22preparationScript%22%3A%22%22%2C%22handlebarsVersion%22%3A%224.7.7%22%7D
